demos/portal-embed-demo/server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const PORTALOG_PORT = 3000;
+const TTT_ARCHIVE_PORT = 3001;
+
+const genServer = (serverName, port, rootHTML) => {
```

Comment:
consistent style with client JS? (e.g. 4sp or 2sp in both places)

Reply:
Formatted to 4sp. Used the VS Code formatter.